server/ws_wrapper.go

```diff
@@ -28,6 +29,11 @@ func (wsw *wsWrapper) Read(p []byte) (n int, err error) {
 			continue
 		}
 
-		return reader.Read(p)
+		if b, err := io.ReadAll(reader); err != nil {
```

Comment:
Why use ReadAll ? If `len(p)` is less than `b`, some bytes may be lost?

Reply:
It didn't handle any request larger than 1024 anyway. You may take a look at `webtty/webtty.go:88`.

Reply:
`Read` is read at most `len(p)` bytes. If you want to `ReadAll`, just call `Read` multiple times until it returns EOF.

Reply:
Maybe we should change the code at https://github.com/tidbcloud/gotty/blob/master/webtty/webtty.go#L90-L98 ?

Reply:
ReadAll and copy is the same as what you just described. You can actually take a look at websocket.Conn.ReadMessage, it is doing the same.

Reply:
`ReadMessage` is `ReadAll` from `NextReader`, but webtty only `Read` 1024 bytes from `NextReader`(See https://github.com/tidbcloud/gotty/blob/master/webtty/webtty.go#L90-L98)

If message is longer than 1024 bytes, this change will `ReadAll` from `NextReader` but only copy 1024 bytes to the buffer.

Reply:
That's the assumption of gotty, right? it didn't want to handle any message that's larger than 1024 which means it didn't meant to read any input larger than 1024.